docs/outlook/smart-alerts-onmessagesend-walkthrough.md

```diff
@@ -258,10 +259,63 @@ In this scenario, you'll add handling for sending a message. Your add-in will ch
 1. In Outlook on Windows, create a new message and set the subject. In the body, add text like "Hey, check out this picture of my dog!".
 1. Send the message. A dialog should pop up with a recommendation for you to add an attachment.
 
-    ![Screenshot of a message window in Outlook on Windows with dialog.](../images/outlook-win-smart-alert.png)
+    ![Dialog recommending the user to include an attachment.](../images/outlook-win-smart-alert.png)
```

Comment:
```suggestion
    ![Screenshot of a dialog recommending that the user include an attachment.](../images/outlook-win-smart-alert.png)
```

Reply:
Thanks for the suggestion. I believe accessibility guides advise to avoid using 'Screenshot of..." or "Image of..." in alt text.